Lib/test/test_argparse.py

```diff
@@ -4958,6 +4958,46 @@ def custom_type(string):
     version = ''
 
 
+class TestHelpUsageLongSubparserCommand(TestCase):
+    """Test that subparser commands are formatted correctly in help"""
+
+    def setUp(self):
+        super().setUp()
+        self.main_program = os.path.basename(sys.argv[0])
+
+    def test_parent_help(self):
+        parent_parser = argparse.ArgumentParser(
+            add_help=False, formatter_class=lambda prog: argparse.RawTextHelpFormatter(prog, max_help_position=50))
```

Comment:
Too long line. PEP 8 recommends to limit all lines to a maximum of 79 characters. Please reformat the code.

For example:
```suggestion
            add_help=False,
            formatter_class=lambda prog:
                argparse.RawTextHelpFormatter(prog, max_help_position=50))
```
You can also use partial():
```suggestion
            add_help=False,
            formatter_class=functools.partial(argparse.RawTextHelpFormatter,
                                              max_help_position=50))
```
Since the same wrapper is used for other parser, you can simply define it as a local function and then reuse it:
```py
        def formatter_class(prog):
            return argparse.RawTextHelpFormatter, max_help_position=50)
```